src/mlpack/methods/time_series_models/simpleexpo.hpp

```diff
@@ -0,0 +1,98 @@
+/**
+ * @file methods/time_series_methods/simpleexpo.hpp
+ * @author Rishabh Bali
+ *
+ * Definition of the SimpleExpo class.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_TIME_SERIES_MODELS_SIMPLEEXPO_MODEL_HPP
+#define MLPACK_METHODS_TIME_SERIES_MODELS_SIMPLEEXPO_MODEL_HPP
+
+#include<mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ts {
+
+/** 
+ * Simple Exponential Smopothing mopdel is one of the simplest time series forcasting methods.
+ * It simply gives more weights to recent observations while forcasting and the 
+ * weights given to older observations exponentially decreases.
+ **/ 
+class SimpleExpo
+{
+ public :
+ /**
+   * Creates the model. and also initializes the parameters.
+   *
```

Comment:
@Ris-Bali I do not see any implementation for this header file? ane you going to submit the implementation?

Reply:
Yes , I am going to submit the implementation;actually I had my Exams going on so thus the delay I'll try to  implement it as soon as possible . 
Thanks

Reply:
Best of luck with your exams.